source/docs/magento2/search-engine.md

```diff
@@ -70,3 +70,87 @@ query Search {
 ```
 
 If you are using the default theme or the theme Chocolatine, the search bar should now be visible.
+
+
+## Algolia
+
+<blockquote class="feature--new">
+  _This feature has been added in Front-Commerce `2.13.0`_
+</blockquote>
+
+### Requirements
+
+On Magento's side, you need to [install and configure the Algolia module for Magento2](https://www.algolia.com/doc/integration/magento-2/getting-started/quick-start/?client=php#installation).
+
+In addition, the attribute `category_ids` must be configured as both an indexed attribute and as a facet:
+
+1. under _Stores > Configuration > Algolia Search > Products_, add `category_ids` as _Searchable_ in the _Attributes_ parameter
+1. under _Stores > Configuration > Algolia Search > Instant Search Result Page_, add `category_ids` in the _Facets_ parameter
+
+You can then run the indexer so that the products are indexed in Algolia's index.
+
+### Front-Commerce configuration
+
+First, you need to make sure the Algolia's search client is installed:
+
+```
+npm i algoliasearch
```

Comment:
**thought:** shouldn't we restrict it to a specific version here (so it matches the one [in optional dependencies](https://gitlab.com/front-commerce/front-commerce/-/blob/b3384a4e567080a0eae5d765f5f06c6be1a28213/package.json#L258))

```suggestion
npm i algoliasearch@4.8
```